trajectory_executor/src/test/trajectory_executor_test_1.cpp

```diff
@@ -53,7 +53,7 @@ TEST_F(TrajectoryExecutorTestSuite, test_runover_shutdown) {
     traj_pub.publish(plan);
 
     std::this_thread::sleep_for(std::chrono::milliseconds(1500));
-    ASSERT_TRUE(recv_sys_alert) << "Failed to receive system shutdown alert message from TrajectoryExecutor node.";
+    ASSERT_FALSE(recv_sys_alert) << "Failed to receive system shutdown alert message from TrajectoryExecutor node.";
```

Comment:
Don't we want this to be `ASSERT_TRUE()` still? I think the concept of this test is that we want to make sure that we do receive the shutdown alert when we run past the trajectory.